### ISSUE TITLE
adempiere vs metasfresh - a first glimpse at SwingUI

### ISSUE THREAD

**Issue by homebeaver** [metasfresh/metasfresh#570]
after a successful build of metasfresh, I made a first glimpse at the swing gui and compare adempiere and metasfresh displaying c_country DE as an example, 
( I use `de.metas.SwingUIApplication_mf15_with_embedded_server.launch`):

![adempierevsmetasfresh](https://cloud.githubusercontent.com/assets/3493126/20102230/a5a7af28-a5c5-11e6-998c-47cc52342124.PNG)

- "fresh" colors 👍 
- new icons 👍 
- IBAN section 👍 
- translations 👎 : worse then the old-adempiere,
espectally for this table/window : c_country has a translation c_country_trl , so if the base language is en and de is a (translated) system lang, we should have c_country.name='Germany' and c_country_trl.name='Deutschland' (currently the base language is set to de)
- CountryCode_3digit 👎 : this is the [ISO A3-code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) - good to have it, but wrong label and I miss
- c_country.numcountrycode in the gui ( 276 for DE , see https://en.wikipedia.org/wiki/ISO_3166-1_numeric )
- DE was set to not active, why?
- ExpressionPostal is not set , why? I think "00000" is a good PLZ-Format. 
- what does the sequence "@CON@" means?
- is "DEU-@P@" ok? Or should it be "D-@P@" 
- to compare I made the window tightened. And the fields on the right side are not formatted well, see red arrow 👎

**Comment by metas-mk** [comment]
@homebeaver thx for your feedback
did a quick first check
- on our current database about translations. sorry, but can't see what you see.
![image](https://cloud.githubusercontent.com/assets/13365687/20129367/7055da36-a64f-11e6-9b92-1d3d095baf7e.png)
- about c_country_trl.name='Deutschland' vs. 'Germany'. In our current metasfresh Database the Base Language is de_DE so it should be correct that c_country_trl.name = 'Deutschland'
![image](https://cloud.githubusercontent.com/assets/13365687/20129431/e956ea60-a64f-11e6-8f90-ea7f0c96300c.png)
- about small window resolution. yes, thats true. we optimized the window sizes for resolution for bigger size. none of our current users has such small screens ;-)

**Comment by homebeaver** [comment]
- I got the db-pgdump from Tobias, see https://gitter.im/metasfresh/metasfresh - Seems, your db has better translations.
- Yes, as I wrote, the base language is set to de. This is no problem for me.
- Not the window resolution, the different width of fields when tightened is what I find faulty. see zipped video:
[2016-11-09_1036.swf..zip](https://github.com/metasfresh/metasfresh/files/580410/2016-11-09_1036.swf.zip)
